### DISCUSSION TITLE
feature request: custom prefixes for sibling nodes

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#685] [Ideas]
Suppose I want to create a list of ideas during a brainstorming session. Right now, I enter, _Idea #1, Idea #2,_ etc. I would like to be able to tell FP that the next sibling node I create should have the same prefix, _Idea #_. Then I could manually enter _1 -, 2 -_ etc. 

The real value of this would be if I had, say 20 ideas, and then wanted, for whatever reason, to change the prefix to _Benefit #_ for a sales pitch or _Memory #_ for a scrapbook, etc.

I know that there are ways, using search and replace, to do something like this now. But such a capability is different from search and replace, at least in my view, and it would significantly make some uses of FP easier.

~ Bal

**Comment by sifran-github** [comment]
You can do that by changing the format of the node to `Idea # %s` (in the latest version: in the format panel, under core text)
You and also add a style that will contain this format, to make it easier to change.

**Reply by wordmuse** [reply]
Very cool - thank you!
~ Bal

**Comment by euu2021** [comment]
I never noticed this template feature! And it would have been useful a couple of times.

I wonder how to make it more easily discoverable?

**Comment by sifran-github** [comment]
Another tip: if you use this format, it also might be helpful to use these options, so you wont have to change the format for each node:

- Format->copy format to a new sibling nodes
- Format->copy format to a new child nodes

Personally, i keep the first option (sibling) checked.

**Reply by wordmuse** [reply]
I keep the first option checked all the time, too. :)
